STORM-IRIT/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ex
+
+filename="${1%.*}"
+
+./pdpcComputeMultiScaleFeatures -i input/$1 -v -o output/$filename
+./pdpcSegmentation -i input/$1 -s output/"${filename}_scales.txt" -f output/"${filename}_features.txt" -o output/$filename -v
+
+mkdir output/range output/pers output/scales
```

Comment:
Try this script with two different input files to check if it works.
Failed for me with
`+ mkdir output/range output/pers output/scales
mkdir: cannot create directory 'output/range': File exists
mkdir: cannot create directory 'output/pers': File exists
mkdir: cannot create directory 'output/scales': File exists
`